### ISSUE TITLE
Bundle Everything is broken

### ISSUE THREAD

**Issue by wollmers** [ThePilgrim/perlcldr#14]
```
$ cpanm Bundle::Locale::CLDR::Everything
--> Working on Bundle::Locale::CLDR::Everything
Fetching http://www.cpan.org/authors/id/J/JG/JGNI/Bundle-Locale-CLDR-Everything-0.29.0.tar.gz ... OK
Configuring Bundle-Locale-CLDR-Everything-v0.29.0 ... OK
==> Found dependencies: Locale::CLDR::Locales::World, Locale::CLDR::Transformations
! Finding Locale::CLDR::Locales::World on cpanmetadb failed.
! Finding Locale::CLDR::Locales::World (0.29.0) on mirror http://www.cpan.org failed.
! Couldn't find module or a distribution Locale::CLDR::Locales::World (0.29.0)
Found Locale::CLDR::Transformations  which doesn't satisfy 0.29.0.
! Installing the dependencies failed: Missing version info for module 'Locale::CLDR::Transformations', Module 'Locale::CLDR::Locales::World' is not installed
! Bailing out the installation for Bundle-Locale-CLDR-Everything-v0.29.0.
```

**Comment by uklauer** [comment]
This appears to have been fixed (for future releases) in commit fa313c0ae0bf2db5a08b3f66b27041b01e5a4f07. I could work around it by directly installing `Bundle::Locale::CLDR::World`.

**Comment by ThePilgrim** [comment]
Reported as fixed
